modules/contact/doc/content/source/postprocessors/MortarContactAreaPostprocessor.md

```diff
@@ -0,0 +1,16 @@
+# MortarContactAreaPostprocessor
+
+`MortarContactAreaPostprocessor` computes the surface area in contact by leveraging
+similar techniques to [VolumePostprocessor.md](VolumePostprocessor), but only
+integrating over elements that have a Lagrange Multiplier variable value above
+a given threshold.
+
+# Description and Syntax
+
```

Comment:
Would you expand out the description section here, include an example of the input file usage, and connect to the class description line from the source file please?

Reply:
I hit a wall on this, @dschwen and @recuero had some good ideas I couldn't quite implement...if someone else wants to take this, that'd be great, otherwise we can close.